src/main/java/mate/academy/spring/boot/intro/model/Book.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.boot.intro.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
Don't use `@Data` for entities, add `@Table` with name
```suggestion
@Getter
@Setter
@Table(name = ...
```